workflows/python/sdk/order-processor/workflow_activities.py

```diff
@@ -0,0 +1,101 @@
+
```

Comment:
Consider naming this file simply workflow.py, since it represents the entire workflow.

Reply:
Done